unicode/eval_engine/services.py

```diff
@@ -17,25 +17,31 @@
 }
 
 
+# Makes the post request to the API and returns an object containing the API output
 def eval_engine(source, lang, test_in, test_out):
-    url = 'http://159.89.93.145:3000/submissions/?base64_encoded=false&wait=true'
+    url = 'http://159.89.93.145:3000/submissions/?base64_encoded=false&wait=true'   # wait=true makes the API call synchronous, halving the number of requests needed

```

Comment:
Can you put this url in the settings file?

Reply:
K